src/appleseed-max-impl/appleseedrenderer/projectbuilder.h

```diff
@@ -75,7 +76,8 @@ struct ObjectInfo
 };
 
 typedef std::map<Object*, std::vector<ObjectInfo>> ObjectMap;
-typedef std::map<std::string, renderer::Entity*> InstanceMap;
+typedef std::map<std::string, renderer::ObjectInstance*> ObjInstanceMap;
```

Comment:
May I suggest not to abbreviate here, for consistency's sake? I.e. `ObjectInstanceMap`